sw/host/tests/chip/rv_dm/src/ndm_reset_req.rs

```diff
@@ -23,6 +23,9 @@ struct Opts {
     timeout: Duration,
 }
 
+// Needs to match util/openocd/target
+const RISCV_IDCODE: u32 = 0x10001cdf;
```

Comment:
I think the ICODE can depend on the part, so on the ES or on real chips it probably won't be this value. Ideally we should make this configurable?

Reply:
There are a few tests currently with this, I'll make the change in a follow-up PR.